### PR TITLE
fix: minor linting

### PR DESCRIPTION
**Pull request by stefl**
## Description

- This addresses some of the items marked as Minor in our Sonar Cloud report
- The idea is to reduce down some of the very small issues so that we can more easily triage the intermediate and major items that have been flagged

Things included:
* Typos
* Options chains
* Nullish coalescing
* Readonly attributes on classes
* Combining type imports
* Empty fragments
* Unnecessary return

### CODE REVIEW

File: apps/nextjs/.storybook/public/mockServiceWorker.js [hunk 1]
```diff
@@ -8,133 +8,133 @@
  * - Please do NOT serve this file on production.
  */
 
-const PACKAGE_VERSION = '2.6.5'
-const INTEGRITY_CHECKSUM = 'ca7800994cc8bfb5eb961e037c877074'
-const IS_MOCKED_RESPONSE = Symbol('isMockedResponse')
-const activeClientIds = new Set()
+const PACKAGE_VERSION = "2.6.5";
```
Comment (stefl) [inline apps/nextjs/.storybook/public/mockServiceWorker.js:11]: These quote changes seem to be automated when saving the file

Comment (codeincontext) [reply apps/nextjs/.storybook/public/mockServiceWorker.js:11]: This file is autogenerated so I assume that we shouldn't be modifying or linting it?

**Review by codeincontext — APPROVED** [review]
Approved, though I'm not sure we care too much about the MSW file

File: apps/nextjs/.storybook/public/mockServiceWorker.js (hunk 1 above)
Comment (stefl) [reply apps/nextjs/.storybook/public/mockServiceWorker.js]: Thanks - I've removed that an added it to ignore config

**Pull request merged**